examples/hosts/app-integration/external-data/tests/externalDataService.test.ts

```diff
@@ -55,14 +55,15 @@ describe("mock-external-data-service", () => {
 		await closeServer(_externalDataService);
 	});
 
-	async function getCurrentExternalData(): Promise<TaskData> {
-		const fetchResponse = await externalDataSource!.fetchData();
+	async function getCurrentExternalData(): Promise<TaskListData> {
+		const taskListId = "task-list-1";
+		const fetchResponse = await externalDataSource!.fetchData(taskListId);
 		const responseBody = JSON.parse(fetchResponse.body.toString()) as Record<
```

Comment:
Consider casting directly to what you want to assume this is (totally fine for demo purposes) or else do type narrowing.

Reply:
Let me get this in a follow up PR.